contrib/completions/meson.build

```diff
@@ -0,0 +1,14 @@
+fish_comp = dependency('fish', required: false)
+
+if get_option('fish-completions')
+	fish_files(
```

Comment:
I think this should be `fish_files = files(…)` instead.

Reply:
Indeed